content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+---
+title: It is not a choice of the messenger
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
```

Comment:
I'd remove either the "just" or the "stood-up and". "I could have just advocated for XMPP" or "I could have stood-up and advocated for XMPP".

Reply:
If it's phrased this way you need a comma after "thought" and quotations around "this alone is not a helpful answer": 'then I thought, "this alone is not a helpful answer"'. Personally I would rephrase so that this isn't necessary.

Also, why is it not a helpful answer, and what does it mean to "only make a short-term decision"? I don't fully understand what this paragraph is trying to say.

Reply:
I intended to say, that people dont think a lot about where to go next. They might just do it, then wonder why it was not a sufficient decision. I also not just wanted to simply argue why XMPP is good, I also wanted to show the general reasons behind it etc.

Maybe that improves a bit:
> I was wondering how to answer this. Obviously, I could have stood-up and advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision about their communication software. On the other hand, I also see they usually do not switch the platform because they made a very evaluated choice for one specific messenger. Many just switch, again and again.

Reply:
> Many people just switch again and again.